Makefile

```diff
@@ -95,16 +95,17 @@ $(NODE_CONTAINER_BIN_DIR)/confd:
 
 # Get the calico-bgp-daemon binary
 $(NODE_CONTAINER_BIN_DIR)/calico-bgp-daemon:
+	$(CURL) -L $(GOBGP_URL) -o $(@D)/gobgp
```

Comment:
A couple of reasons.
-  Consistency with BIRD, we include the CLI to aide debugging.
-  We don't currently have a working status command that is checked in do we .. and if we do, are we ST-ing it, because we should be.
-  It's useful if you want to poke around (like I needed to last night).

I guess it boils down to what % it adds to the image.  I wonder if we should produce a gobgp and a BIRD calico/node image so it has reduced size